napari/layers/_image_layer/_constants.py

```diff
@@ -0,0 +1,30 @@
+from enum import auto
+
+from ...util.misc import StringEnum
+
+
+class Interpolation(StringEnum):
+    """INTERPOLATION: Vispy interpolation mode.
+
+    The spatial filters used for interpolation are from vispy's
+    spatial filters. The filters are built in the file below:
+
+    https://github.com/vispy/vispy/blob/master/vispy/glsl/build-spatial-filters.py
+    """
+    BESSEL = auto()
+    BICUBIC = auto()
+    BILINEAR = auto()
+    BLACKMAN = auto()
+    CATROM = auto()
+    GAUSSIAN = auto()
+    HAMMING = auto()
+    HANNING = auto()
+    HERMITE = auto()
+    KAISER = auto()
+    LANCZOS = auto()
+    MITCHELL = auto()
+    NEAREST = auto()
+    QUADRIC = auto()
```

Comment:
drop quadric and sinc as described in #132

Reply:
Great point. Done!